core/types/transaction_marshalling_test.go

```diff
@@ -2,11 +2,28 @@ package types
 
 import (
 	"encoding/json"
+	"math/big"
 	"testing"
 
+	"github.com/ethereum/go-ethereum/common"
 	"github.com/stretchr/testify/require"
 )
 
+func Test_Transaction_UnmarshalJson_Deposit(t *testing.T) {
```

Comment:
```suggestion
func TestTransaction_UnmarshalJson_Deposit(t *testing.T) {
```
nit on go uses either PascalCase or camelCase. I'm kinda surprised the test below uses an underscore, most other tests in geth don't use them.

Reply:
I added the test below in another PR. :) I think it was based on GoLand suggested naming... Will fix them all up in one commit.